test/verify/run-tests

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python3
```

Comment:
So there is only one verify file, so this is not needed. But I guess you plan to extend another tests in the future, right?

Reply:
Yes, more files are on the way. :)